src/ValidationContext.php

```diff
@@ -243,16 +243,11 @@ public function getAttributeLabel(): ?string
     /**
      * Get translated attribute name.
      *
-     * @return string|null Translated attribute name. `null` if a single value is validated and there is nothing
-     * to translate.
+     * @return string Translated attribute name. `value` if a single value is validated and a label is not set.
      */
-    public function getTranslatedAttribute(): ?string
+    public function getTranslatedAttribute(): string
     {
-        if ($this->attribute === null) {
-            return null;
-        }
-
-        $label = $this->attributeLabel ?? $this->attribute;
+        $label = $this->attributeLabel ?? $this->attribute ?? 'value';
```

Comment:
What if `int` attribute names will be used? Will the result be like "0 is not valid."? Here the attribute name can be confused with actual value. Maybe turn it to countable items then? Like "1th item is not valid". Can be discussed in a separate issue and improved in a separate PR.

Reply:
The value cannot be `int`. For `int` keys attributes is `null`.

Reply:
Sorry, didn't get you. Could you demonstrate it with an example?

Reply:
See here: https://github.com/dood-/validator/blob/enh-attribute-name-in-messages/src/Validator.php#L134-L142